src/components/Uploader/ImageQualityGuide.js

```diff
@@ -0,0 +1,164 @@
+import { h, Component } from 'preact'
+import classNames from 'classnames'
+import { localised } from '../../locales'
+import { trackComponentAndMode } from '../../Tracker'
+import { isDesktop, addDeviceRelatedProperties } from '~utils'
+import UploadError from './Error'
+import { validateFileTypeAndSize } from '~utils/file'
+import { randomId } from '~utils/string'
+import theme from '../Theme/style.css'
+import style from './style.css'
+import PageTitle from '../PageTitle'
+import Button from '../Button'
+import CustomFileInput from '../CustomFileInput'
+
+const UploadButton = ({ text }) => (
+  <Button
+    variants={['centered', 'primary']}
+    className={classNames(style.button, style.passportUploadButton)}>
+    {text}
+  </Button>
+)
+
+class ImageQualityGuide extends Component<Props, State> {
+  static defaultProps = {
+    onUpload: () => {}
+  }
+
+  setError = (name) => this.setState({ error: { name } })
+
+  createCapture = (file) => {
+    const payload = { blob: file, sdkMetadata: { captureMethod: 'html5' } }
+    const { documentType, actions, mobileFlow } = this.props
+    const documentCaptureData = {
+      ...payload,
+      sdkMetadata: addDeviceRelatedProperties(payload.sdkMetadata, mobileFlow),
+      method: 'document',
+      side: 'front',
+      documentType,
+      id: randomId(),
+    }
+    actions.createCapture(documentCaptureData)
+  }
+
+  handleFileSelected = (file) => {
+    const error = validateFileTypeAndSize(file)
+    if (error) {
+      this.setError(error)
+    } else {
+      this.createCapture(file)
+      this.props.nextStep()
+    }
+  }
+
+  render() {
+    const { translate } = this.props
+    const { error } = this.state
+    const imgAltText = translate('image_quality_guide.img_alt_text')
+    const buttonText = translate('image_quality_guide.next_step')
+    return (
+      <div className={theme.fullHeightContainer}>
+        <PageTitle
+          title={translate('image_quality_guide.title')}
+          subTitle={translate('image_quality_guide.sub_title')}
+        />
+        <div
+          className={classNames(theme.thickWrapper, style.imageQualityGuide)}
```

Comment:
You should probably use the `theme.scrollableContent` here, once you reorganise the structure of this page as per my other comment

Reply:
My concern about using `theme.scrollableContent` here also is that we might get a double scrollbar on smaller screens since the main parent container of this already uses that class.

Reply:
You're right, you don't need to use scrollableContent here. Like we do on other screens, I think the images should just resize on small screens.